src/js/lib/monster.ui.js

```diff
@@ -1465,6 +1466,28 @@ define(function(require) {
 			return form.valid();
 		},
 
+		/**
+		 * Convert the 'target' in to Markdown editor
+		 * @param target - mandatory jQuery object
+		 * @param [options] - optional object to overwrite the default configs
+		 */
+		simplemde: function($target, options) {
+			if (!($target instanceof $)) {
+				throw TypeError('"$target" is not a jQuery object');
+			}
+
+			if (!_.isUndefined(options) && !_.isPlainObject(options)) {
+				throw TypeError('"options" is not a plain object');
+			}
+
+			new simplemde(_.merge({
```

Comment:
It's always a good idea for `monster.ui` helpers to return the instance created by the constructor to allow chaining.